react-charts/area-chart/step1.md

```diff
@@ -3,8 +3,23 @@ This provides a collection of React based components you can use to build Patter
 
 PatternFly React charts are composed using separate components, so features are more easily customizable. 
 In this course, we're going to build a PatternFly area chart together - starting with a simple chart, adding multiple datasets, tooltips, axis labels, a legend, and concluding by changing the theme color.
-You'll learn how to use PatternFly React chart components together to build a consistent user experience.
+
+You will learn how to use PatternFly React chart components together to build a consistent user experience.
 
 The image below is what we'll end up with when we are done.
 
 <img src="area-chart/assets/final.png" alt="Completed chart component" style="box-shadow: rgba(3, 3, 3, 0.2) 0px 1.25px 2.5px 0px;" />
+
+## Task
+
+1) <strong> Navigate to the `src` folder and open `src/app.css`{{open}}.</strong>
+
+2) Next you will <strong>add some CSS </strong>to define the overall chart size.
+
+3) <strong>Copy the following code into the app.css file, replacing all of the content there:</strong>

```

Comment:
When we can, let's use "all" instead of "all of." This way, we're sticking with a simpler and more concise style. Of course, sometimes it's necessary to use "all of." But we should omit the "of" whenever we can provided we're still making sense.